python/dgl/nn/mxnet/conv/graphconv.py

```diff
@@ -109,8 +110,14 @@ def forward(self, graph, feat, weight=None):
         ----------
         graph : DGLGraph
             The graph.
-        feat : mxnet.NDArray
-            The input feature.
+        feat : mxnet.NDArray or pair of mxnet.NDArray
+            If a single tensor is given, represents the input feature of shape :math:`(N, D_{in})`
+            where :math:`D_{in}` is size of input feature, :math:`N` is the number of nodes.
+            If a pair of tensors are given, the pair must contain two tensors of shape
+            :math:`(N_{in}, D_{in_{src}})` and :math:`(N_{out}, D_{in_{dst}})`.
+
+            Note that in the special case of graph convolutional networks, if a pair of
+            tensors is given, the latter element will not participate in computation.
```

Comment:
Why there will be a pair of tensor?

Reply:
Because one may still want to train GCN stochastically, which may feed the GCN module with a block (i.e. bipartite graph).